src/execution/standard.ts

```diff
@@ -60,57 +60,61 @@ export class StandardScriptExecution extends BaseExecution<StandardScriptConfig>
   }
 
   protected async _execute(): Promise<ExecutionResult> {
-    this._ensureState('before-running');
-
-    const dependencyFingerprints = await this._executeDependencies();
-    if (!dependencyFingerprints.ok) {
-      dependencyFingerprints.error.push(this._startCancelledEvent);
-      return dependencyFingerprints;
-    }
-
-    // Significant time could have elapsed since we last checked because our
-    // dependencies had to finish.
-    if (this._shouldNotStart) {
-      return {ok: false, error: [this._startCancelledEvent]};
-    }
+    try {
```

Comment:
Note this is not a very well formatted diff. It's all indentation inside the `try`.